engine/telegram.js

```diff
@@ -124,13 +124,6 @@ module.exports = (config) => {
         //         }
         //     })
         // },
-
-        typingOn: () => {
-            return message({text: "Typing..."})
-        },
-        typingOff: () => {
-            return senderAction(typing_off_str)
-        }
```

Comment:
This doesn't seem right: typingOn is needed, you can't simply drop it, we discussed that typingOff can be a nop for now, so that you don't have to waste time on it (next message will cancel typing indication anyways)